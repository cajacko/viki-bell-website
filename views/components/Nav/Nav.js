/* @flow */

import React from 'react';

const Nav = () => (
  <nav>
    <ul>
      <li><a>Burger</a></li>
      <li><a>Posts</a></li>
      <li><a>About</a></li>
      <li><a>Life</a></li>
      <li><a>Food</a></li>
      <li><a>Travel</a></li>
    </ul>

    <ul>
      <li><a>Twitter</a></li>
      <li><a>Instagram</a></li>
      <li><a>Facebook</a></li>
      <li><a>More Social</a></li>
    </ul>
  </nav>
);

export default Nav;
