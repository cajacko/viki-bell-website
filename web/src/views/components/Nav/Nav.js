/* @flow */

import React from 'react';

const Nav = () => {
  const navItems = [
    {
      title: 'Home',
      url: '/',
    },
    {
      title: 'Item 1',
      url: '/post/1',
    },
    {
      title: 'Item 2',
      url: '/post/2',
    },
    {
      title: 'Item 3',
      url: '/post/3',
    },
    {
      title: 'Item 4',
      url: '/post/4',
    },
  ];

  return (
    <ul>
      {
        navItems.map(({ url, title }) => (
          <li key={url}><a href={url}>{title}</a></li>
        ))
      }
    </ul>
  );
};

export default Nav;
