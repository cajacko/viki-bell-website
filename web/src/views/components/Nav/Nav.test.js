import React from 'react';
import Nav from 'components/Nav/Nav';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Nav />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
