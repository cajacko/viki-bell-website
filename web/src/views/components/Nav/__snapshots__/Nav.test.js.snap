// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <ul>
    <li>
      <a>
        Burger
      </a>
    </li>
    <li>
      <a>
        Posts
      </a>
    </li>
    <li>
      <a>
        About
      </a>
    </li>
    <li>
      <a>
        Life
      </a>
    </li>
    <li>
      <a>
        Food
      </a>
    </li>
    <li>
      <a>
        Travel
      </a>
    </li>
  </ul>
  <ul>
    <li>
      <a>
        Twitter
      </a>
    </li>
    <li>
      <a>
        Instagram
      </a>
    </li>
    <li>
      <a>
        Facebook
      </a>
    </li>
    <li>
      <a>
        More Social
      </a>
    </li>
  </ul>
</nav>
`;
